js/views/PageNav.js

```diff
@@ -4,7 +4,7 @@ import { View } from 'backbone';
 import loadTemplate from '../utils/loadTemplate';
 import app from '../app';
 import $ from 'jquery';
-import SettingsModal from './modals/settings/Settings';
+import SettingsModal from './modals/Settings/Settings';
```

Comment:
Re: the Settings folder, it's capitalized in master, when I changed it to lower case (here, and changed the folder name to lower case) Travis threw an error.